### ISSUE TITLE
[ldd-survey] Limiting_Magnitudes description is circular

### ISSUE THREAD

**Issue by wkiri** [pds-data-dictionaries/PDS4-LDD-Issue-Repo#195]
**Describe the issue identified (if applicable)**
The Limiting_Magnitudes definition is "The limiting magnitude of the image".  

**Describe the solution you'd like**
A clear and concise description.

**LDD Dictionary Version**
1.0.1.0

**PDS4 IM Version**
1.H.0.0

**Need-by Date**
Not critical.

**Comment by wkiri** [comment]
Per @jstone-psi , there may be other attribute or class descriptions that would benefit from expansion as well.

**Comment by rgdeen** [comment]
sure, but limiting magnitude is also a well-known term in the astronomical community.  Is it the job of the PDS DD to provide remedial instruction in common terms for a given domain?  Although I certainly don't mind better descriptions of things (in general it's pretty poor), we can get into absurdities if taken to extremes.

Note that I don't actually object to augmenting this particular definition, just a cautionary note that it can be taken too far quite easily.  Do we need to define "magnitude" as part of this?  "Image"?  Where do you draw the line?

**Comment by wkiri** [comment]
@rgdeen Of course, we don't want to go crazy on this, but it's helpful to try to step outside the term to define it.  In some cases, the person reading the definition might be trying to decide if this namespace is relevant for their data, and including more details in the definitions makes them more accessible.  As one example: https://en.wikipedia.org/wiki/Limiting_magnitude
